Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Remove path declaration from the initscript
 All utilities reside on standard paths. Furthermore, the declaration of
 /usr/sbin makes lintian complain about a missing dependency on $remote_fs,
 which is strictly not necessary.

Last-Update: 2014-06-13
Forwarded: no
--- a/scripts/drbd
+++ b/scripts/drbd
@@ -31,8 +31,6 @@
 UDEV_TIMEOUT=10
 ADD_MOD_PARAM=""
 
-PATH=/usr/sbin:/sbin:$PATH
-
 if [ -f $DEFAULTFILE ]; then
   . $DEFAULTFILE
 fi
